src_assets/common/assets/web/Checkbox.vue

```diff
@@ -0,0 +1,108 @@
+<script setup>
+const model = defineModel({ required: true });
+const slots = defineSlots();
+const props = defineProps({
+  class: {
+    type: String,
+    default: ""
+  },
+  desc: {
+    type: String,
+    default: null
+  },
+  id: {
+    type: String,
+    required: true
+  },
+  label: {
+    type: String,
+    default: null
+  },
+  localePrefix: {
+    type: String,
+    default: "missing-prefix"
+  },
+  checkedByDef: {
+    type: Boolean,
+    default: false
+  },
+  uncheckedByDef: {
+    type: Boolean,
+    default: false
+  }
+});
+
+// Add the mandatory class values
+const extendedClassStr = (() => {
+  let values = props.class.split(" ");
+  if (!values.includes("form-check")) {
+    values.push("form-check");
+  }
+  return values.join(" ");
+})();
+
+// Determine the true/false values for the checkbox
+const checkboxValues = (() => {
+  const mappedValues = (() => {
+    // Try literal values first
+    let value = model.value;
+    if (value === true || value === false) {
+      return [true, false];
+    }
+    if (value === 1 || value === 0) {
+      return [1, 0];
+    }
+
+    // Try mapping strings next (first in the list will be used as fallback)
+    const stringPairs = [
+      ["true", "false"],
+      ["1", "0"],
+      ["enabled", "disabled"],
+      ["enable", "disable"],
+      ["yes", "no"],
+      ["on", "off"]
+    ];
```

Comment:
Is there a way we can migrate values over to true/false whenever they make the next save?

Reply:
Unfortunately, it is not that trivial:

1. Each page handles it's own configuration and has it's own "default" values. Ideally this place (and only) needs to be adapted to migrate the values, which leads me to the next point.
2. Sunshine gives the config as a `map<string, string>` to the UI, whereas the default values from the UI can also be boolean literals (not strings). However, then the literals are saved as strings!? It's a mess...

Based on those 2 points I would strongly suggest against trying Checkbox do any modification (if that was your intention) and properly cleanup the deserialize/serialize code for each page in the UI, but not now plz...